extensions/powershell/resources/built-time-cmdlets.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Why do we need an .SLN at all?

Reply:
Because when you save in VS, it creates it if it isn't there. So, it just makes sense to be there and use it to open for editing. I've been using this (not checked in) for a while but it makes sense so others can edit the cmdlets easily.